lang/cs/Org.Apache.REEF.Client/AzureBatch/Service/AzureBatchService.cs

```diff
@@ -39,10 +41,24 @@ public sealed class AzureBatchService : IDisposable
         public string PoolId { get; private set; }
 
         private BatchClient Client { get; set; }
+        private ContainerRegistryProvider ContainerRegistryProvider { get; set; }
+        internal ITcpPortProvider PortProvider { get; set; }
+        private ICommandBuilder CommandBuilder { get; set; }
```

Comment:
PortProvider is set in constructor and also setter. I think we should only allow one of them.

Reply:
Changed to private.

---
In reply to: [201092944](https://github.com/sharathmalladi/reef/pull/5#discussion_r201092944) [](ancestors = 201092944)